source/code-snippets/monitoring/cpm-status.js

```diff
@@ -0,0 +1,32 @@
+function connectionPoolStatus(client) {
+  let checkedOut = 0;
+
+  function onCheckout() {
+    checkedOut++;
+  }
+
+  function onCheckin() {
+    checkedOut--;
+  }
+
+  function onClose() {
+    client.removeListener('connectionCheckedOut', onCheckout);
+    client.removeListener('connectionCheckedIn', onCheckin);
+
+    checkedOut = NaN;
+  }
+
+  // Increases count of active connection pool threads when connectionCheckedIn event is triggered
```

Comment:
I: since this line calls the `onCheckin()` function, which _decreases_ the checkedOut count, it seems a little misleading to say that this is increasing a count of active connection pool threads. Maybe you could reword to:
```suggestion
  // Decreases count of connections checked out of the pool when connectionCheckedIn event is triggered
```